test/unit_tests/modules/util/dom-traverse-spec.js

```diff
@@ -35,3 +35,87 @@ describe( 'Dom Traverse queryOne', function() {
     expect( query ).to.equal( null );
   } );
 } );
+
+describe( 'Dom Traverse getSiblings()', function() {
+  jsdom();
+
+  before( function() {
+    document.body.innerHTML =
+      '<div class="div-1"></div><div class="div-2"></div>';
+  } );
+
+  it( 'should return an array with a single sibling',
+    function() {
+      var elem = document.querySelector( '.div-1' );
+      var siblings = domTraverse.getSiblings( elem, 'div' );
+
+      expect( siblings.length ).to.equal( 1 );
+      expect( siblings[0].className ).to.equal( 'div-2' );
+    }
+  );
+} );
+
+describe( 'Dom Traverse not()', function() {
+  jsdom();
+
+  before( function() {
+    document.body.innerHTML =
+      '<div class="div-1"></div><div class="div-2"></div>';
+  } );
+
+  xit( 'should return an array with the item that wasn’t excluded',
```

Comment:
Oh duh! Thank you for catching that. I've updated the PR with your suggestions minus the extra return.